tests/unittests/tests-hashes/tests-hashes-sha256.c

```diff
@@ -134,6 +134,23 @@ static const unsigned char hlong_sequence[] =
                                     0x6a, 0x78, 0x21, 0x73, 0x54, 0x89, 0x61, 0x85,
                                     0xb1, 0x4a, 0x3a, 0x84, 0xf7, 0xcd, 0x80, 0x66};
 
+/**
```

Comment:
Why the new test cases, was there a blind spot somewhere? Otherwise I have the impression that the commit targets adding tests for the `hash_wrapper`?

Reply:
These are standard test vectors by NIST which I added because most of the others only deal with online-generated test vectors. Other than that, I added the `hash_wrapper` test execution as suggested by @mtausig.

Reply:
Does it make sense to add a reference of that somewhere?

```
NIST test vectors 1& 2  (see FIPS PUB 180-2 Appendix B)
```

or renaming the test vectors to ` h_nist_1`, something to make the source clear?

I would have added both changed (new vectors + test wrapper) in separate commits. But I won't insist on that.